From d55bafabde8dd6093b1f88039098a4b457a2411c Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 14 Aug 2020 02:39:52 -0700
Subject: [PATCH 1516/1852] dt-bindings: dma: xilinx_frmbuf: Fix Y10 v4l2 pixel
 format

commit bb12ab93ac3238e90d74e555f245875a08ac519a from
https://github.com/Xilinx/linux-xlnx.git

The Y10 format for Framebuffer Write IP write 4 bytes with 3 pixels
information in the memory. This is represented by the custom
V4L2_PIX_FMT_XY10 and not V4L2_PIX_FMT_Y10 which saves only 1 pixel in
2 bytes in the memory.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
index 484389930dca..35fea95bbbcf 100644
--- a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
+++ b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
@@ -76,7 +76,7 @@ UYVY8		uyvy		V4L2_PIX_FMT_UYVY	DRM_FORMAT_UYVY
 YUVA8		avuy8888	<not supported>		DRM_FORMAT_AVUY
 YUVX10		yuvx2101010	V4L2_PIX_FMT_XVUY10	DRM_FORMAT_XVUY2101010
 Y8		y8		V4L2_PIX_FMT_GREY	DRM_FORMAT_Y8
-Y10		y10		V4L2_PIX_FMT_Y10	DRM_FORMAT_Y10
+Y10		y10		V4L2_PIX_FMT_XY10	DRM_FORMAT_Y10
 Y_UV8		nv16		V4L2_PIX_FMT_NV16	DRM_FORMAT_NV16
 Y_UV8		nv16		V4L2_PIX_FMT_NV16M	DRM_FORMAT_NV16
 Y_UV8_420	nv12		V4L2_PIX_FMT_NV12	DRM_FORMAT_NV12
-- 
2.31.1

